src/library/PiaFormat.ts

```diff
@@ -0,0 +1,94 @@
+import dayjs from "dayjs";
+
+enum PIATypesEnum {
+  piaString = "string",
+  piaDate = "date",
+  piaMonthYear = "monthyear",
+  piaInt = "int",
+}
+
+const piaLineParsers = {
+  "01": (contents: string) => {
```

Comment:
For each of these parsers, it would be better to explicitly type the return value. The simplest approach is to say `(contents: string): Partial<PiaData> => {...}`. That way, Typescript will make sure that we return an object with keys from `PiaData`.

Looking at the PIA spec, this map-line-id-to-parser approach will get trickier since it seems like some of the lines are parts of ranges that would all dump data into the same field (e.g., benefit increases). Just a heads up, think it's fine for now.